09-schema/00-overview.md

```diff
@@ -57,26 +73,29 @@ Types constitute the core of your schema. They provide the necessary vocabulary
 
 __Entities__ are the main actors in our domain. These are usually the type of things we want to know about. Entity types provide means of classifying the objects in our domain.
 
-__Relationships__ are things that connect other concepts. Each relationship can connect a number of things, as specified in your schema. The character of participation in a relationship is characterised by roles that can be played in that relationship. Each relationship is required to
+__Relations__ connect other things together. Each relation can connect a number of things.
+The character of participation in a relation is characterised by a __role__ that can be played in that relation. Each relation is required to
```

Comment:
A thing's participation in a relation...
This way you don't have _character_ and _characterised_ in the same sentence